apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderContext.cs

```diff
@@ -60,22 +58,12 @@ internal sealed class TraceHeaderContext
         /// </summary>
         public static TraceHeaderContext Create(string traceId, ulong? spanId, bool shouldTrace) =>
             new TraceHeaderContext(traceId, spanId, shouldTrace);
-        
 
         /// <summary>
-        /// Creates a <see cref="TraceHeaderContext"/> from an <see cref="HttpRequest"/>. 
+        /// Creates a <see cref="TraceHeaderContext"/> from a header. 
         /// </summary>
-        public static TraceHeaderContext FromRequest(HttpRequest request)
+        public static TraceHeaderContext FromHeader(string header)
```

Comment:
It might be helpful to have a \<param\> tag explain a bit more about the header value.